app/models/participant.rb

```diff
@@ -0,0 +1,7 @@
+class Participant < ApplicationRecord
+  #TODO: Add relationships
```

Comment:
Let's try to keep comments (that don't explain/document code) out of master for now! I would delete the TODO & relationship comments.